schema/tables.sql

```diff
@@ -1478,6 +1516,30 @@ CREATE TABLE `ContainerReport` (
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `ContainerType`
+--
+
+DROP TABLE IF EXISTS `ContainerType`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `ContainerType` (
```

Comment:
I think we might want a subtype too. At the moment the UI does some horrible string check to know whether to show the plate, puck, or pcr strip interface. Subtypes might be puck, plate, scm? We have to work out how to display these SCM containertypes. For PCR we actually just use the plate viewer with a single row.